packages/api/src/auth/decoders/ethereum.ts

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken'
+
+export const ethereum = (token: string) => {
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
@Tobbe we're proposing to do away with this env check since NODE_ENV isn't present in the Netlify function runtime, correct?

And just raising an Error is the secret is undefined?

Reply:
Correct.

I noticed that just throwing an error will give the frontend a HTTP 500 error code, but the thrown error-text is swallowed somewhere. So I added a `console.error` printout as well to be a little more helpful to the developer.

Reply:
Updated to match the supabase example!